### PR TITLE
Adding chunky-metro repos

### PR DESCRIPTION
**Pull request by rhizanthemum**

### CODE REVIEW

File: data/ecosystems/s/solana.toml [hunk 1]
```diff
@@ -16733,6 +16733,15 @@ url = "https://github.com/chungquantin/solide"
...
 url = "https://github.com/chungquantin/typescript-sdk"
 
+[[repo]]
+url = "https://github.com/chunky-metro/chunkybot"
+
+[[repo]]
+url = "https://github.com/chunky-metro/december"
+
+[[repo]]
+url = "https://github.com/chunky-metro/jupiter-ag-sdk"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:16743]: missing

File: data/ecosystems/s/solana.toml [hunk 2]
```diff
@@ -16733,6 +16733,15 @@ url = "https://github.com/chungquantin/solide"
 [[repo]]
 url = "https://github.com/chungquantin/typescript-sdk"
 
+[[repo]]
+url = "https://github.com/chunky-metro/chunkybot"
+
+[[repo]]
+url = "https://github.com/chunky-metro/december"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:16740]: empty? can't see any code

File: data/ecosystems/s/solana.toml [hunk 3]
```diff
@@ -16733,6 +16733,15 @@ url = "https://github.com/chungquantin/solide"
 [[repo]]
 url = "https://github.com/chungquantin/typescript-sdk"
 
+[[repo]]
+url = "https://github.com/chunky-metro/chunkybot"
```
Comment (eherrerosj) [inline data/ecosystems/s/solana.toml:16737]: empty? can't see any code

**Review by eherrerosj — CHANGES_REQUESTED** [review]
how are these contributing to Solana

**Comment by rhizanthemum** [comment]
@eherrerosj these are all very early WIP but I removed the other two for now

**Comment by jubos** [comment]
Manually merged here: [b2c8993](https://github.com/electric-capital/crypto-ecosystems/commit/b2c8993c97025e8c02f1aa95ed2a2f591cbcf83e)

**Pull request closed without merging**